bundles/auth/org.eclipse.smarthome.auth.oauth2client/src/main/java/org/eclipse/smarthome/auth/oauth2client/internal/cipher/NullStorageCipher.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2014,2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.smarthome.auth.oauth2client.internal.cipher;
+
+import org.eclipse.smarthome.core.auth.client.oauth2.StorageCipher;
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * This is a NullCipher (no encryption/decryption - identity) implementation of
+ * {@link org.eclipse.smarthome.core.auth.client.oauth2.StorageCipher}
+ *
+ * This should only be used for development.
+ *
+ * @author Gary Tse - Initial Contribution
+ *
+ */
+@Component(name = "org.eclipse.smarthome.auth.oauth2client.internal.cipher.NullStorageCipher", property = "service.ranking=0")
```

Comment:
By default the name will be the fully-qualified classname. For simplicity, you can just name it something smaller. For example, `NullStorageCipher`

Reply:
And the service ranking is also by deafult set to 0 if nothing is mentioned.

Reply:
Originally I wanted to emphasis the service name, and the service-ranking in here -- because I wanted NullStorageCipher to have the default 0 but the SymmetricKeyCipher have a higher ranking.  Since there is a "getUniqueCipherId" these 2 have no purpose.